api/graphql/schema.graphql

```diff
@@ -524,6 +532,10 @@ type Team {
   departments: [Department] @belongsToMany
   contactEmail: Email @rename(attribute: "contact_email")
   pools: [Pool] @hasMany @can(ability: "viewAny")
+  # teams and roles: built-in
+  users: [UserPublicProfile!] @morphToMany
+  # teams and roles: custom pivot model
+  roleAssignments: [RoleAssignment!] @hasMany
```

Comment:
I'm wondering if we want to expose the users relationship at all, since it's kind of redundant with rolesAssignements. (We could consider it to have some worth as a convenience method, since it saves the client from doing a bit of processing if they just want the list of unique users. Actually, question, does it handle filtering out duplicate users?)

The other thing is these fields definitely need `@can` directives, since we have different permissions for viewing teams, and viewing team members. I figure both of these would be protected with `@canOnParent(ability: 'viewTeamMembers')`.

Reply:
Removed in [remove extra relationships](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/69500c02baa6a090cdde1213656bb8672c78251c).

Reply:
Added viewTeamMembers guard in [Update guards](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/804da4ff4cf20bfa9513bcce20f8d8c1b32a86cc).